en/docs/get-started/quick-start-guide.md

```diff
@@ -109,90 +101,87 @@ In this guide, we will add a new resource to the API and simple business logic:
 
      ![Configure Variable](../assets/img/get-started/configure-variable.png){.cInlineImage-full}
 
-9. Click **Save**.
+8. Click **Save**.
 
-10. Let's return the message. Click **+** below the variable we created on the low-code diagram and select **Return**. 
+9. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
    ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
 
-    ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
+   ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
    
-11. Enter `message` as the value in the **Return Expression** text box.
-    
-12. Click **Save**.
+10. Click **Save**.
 
 !!!info
-    Notice that you can click on the **</>** icon on the top left corner of the editor to open the code-view at any given time. You can choose to develop on the low-code or pro-code or both simultaneously. 
-    ![Edito View](../assets/img/get-started/low-code-pro-code-side-by-side.png){.cInlineImage-full}
+    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can choose to work in the low-code view or pro-code view, or both simultaneously.    
+    ![Editor View](../assets/img/get-started/low-code-pro-code-side-by-side.png){.cInlineImage-full}
 
 ### Step 2: Commit Changes to GitHub
 
-Choreo maintains source code in a private GitHub repository. When you create a component in Choreo, it creates it in the private-GitHub repository associated with your account. Therefore, when you change the API Implementation, you need to commit and push your changes. 
+Choreo maintains source code in a private GitHub repository. When you create a component, Choreo creates it in the private-GitHub repository associated with your account. Therefore, you must commit and push your changes if you want to change the API Implementation.
```

Comment:
if you want to change the API Implementation -> if you want to persist and changes needs to be reflected Is the meaning we need to give out.